vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -1132,6 +1132,15 @@ void WindowsTextInputComponentView::updateLayoutMetrics(
 
   if (newWidth != m_imgWidth || newHeight != m_imgHeight) {
     m_drawingSurface = nullptr; // Invalidate surface if we get a size change
+
+    // call onContentSizeChange event for multiline TextInput
+    if (m_eventEmitter && !m_comingFromJS && m_multiline) {
+      auto emitter = std::static_pointer_cast<const facebook::react::WindowsTextInputEventEmitter>(m_eventEmitter);
+      facebook::react::WindowsTextInputEventEmitter::OnContentSizeChange onContentSizeChangeArgs;
```

Comment:
I dont think this is the content size.  This is the size of the textinput - But I think onContentSize is supposed to be the size of the text content.  So if there is more text that fits within the TextInput bounds, and the TextInput starts scrolling, then onContentSize would report a value larger than the layoutMetrics.frame.size.

Reply:
Yeah, verified in Expo Snack - In 2 line multiline textinput, as I add more and more lines of text the contentSize increases even through the textinput itself does not increase in size.